packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewManager.java

```diff
@@ -357,6 +358,9 @@ public void receiveCommand(ReactViewGroup root, int commandId, @Nullable Readabl
         handleSetDestinations(root, args);
         break;
       }
+      case CMD_REQUEST_TV_FOCUS:
+        root.requestFocus();
+        break;
```

Comment:
I believe this is not related to new/old arch difference, to be accurate. There used to be `integer` commands and we were handling them the way you did it here but they're deprecated for a while. Commands are sent as string from JS side and there shouldn't be a way to send `integer` commands. The changes in this file should be safe to remove

Related PR: https://github.com/facebook/react-native/commit/3cae6fa950eed54bccccf61a28c0e85d5a004c6c#diff-71bc3f19ebb0dbc0c2f2c7c2fc24368d9bcf26118b43da76b1493a71f9375860R35